settings.gradle

```diff
@@ -177,5 +177,4 @@ include "beam-test-infra-metrics"
 project(":beam-test-infra-metrics").dir = file(".test-infra/metrics")
 include "beam-test-tools"
 project(":beam-test-tools").dir = file(".test-infra/tools")
-include "beam-test-jenkins"
```

Comment:
I guess we still want to keep this module?

Reply:
Oh my bad, I might have accidentally deleted it.